rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -1511,6 +1513,54 @@ public static abstract class VulnerabilityApiDTOMixin extends VulnerabilityApiDT
             abstract public String getId();
         }
 
+        @JsonInclude(JsonInclude.Include.NON_EMPTY)
+        @JsonIgnoreProperties({
+                "id",
```

Comment:
Without the `id`, how could one perform the operation to change the vulnerability status? Is there a separate endpoint handling this?

![image](https://github.com/user-attachments/assets/ffd1a316-fb00-4296-becf-3b851d5adc2d)

Reply:
Hi @GMishx, I have removed id from response because we don't have any operations related to vulnerability on summary page. For individual project's vulnerability we have endpoint to perform the required operation.

Reply:
So, if someone has to update the vulnerability status, they directly use the vulnerability id?

But how would they receive it?

Reply:
For vulnerability Summary I have created a separate class, VulnerabilitySummMixin and in vulnerability summary we don't perform any action therefore id is not required.